src/components/Core/style.scss

```diff
@@ -59,51 +59,52 @@
   --mercury-font-family-alt: 'PP Monument Extended, sans-serif';
```

Comment:
Are there any changes here or is this just the result of no prettier in this branch?

Reply:
I just updated the colors hex to uppercase letters

Reply:
I just updated the colors hex to uppercase letters

Reply:
Hmm ok - If we want that approach, we should add a eslint/prettier rule for it. I'll see what we can do about it